bindings/py/cpp_src/plugin/unittest/PyRegionTest.cpp

```diff
@@ -409,7 +378,23 @@ void testWriteRead() {
   }
 }
 
-int realmain(bool leakTest) {
+
+TEST(PyRegionTest, testAll) { //TODO former main method, could be splitted into separate tests
+
+  // This isn't running inside one of the SWIG modules, so we need to
```

Comment:
SWIG?   we need to initialize python because we are originating an execution of python code from within C++.

Reply:
i just copied the commet from the test I moved here. Should I change it somehow? The SWIG also hit me in the eyes.